Map/MapMovement.gd

```diff
@@ -0,0 +1,47 @@
+class_name MapMovement extends Resource
+
+
+static func get_accessible_room_positions_by_player(position: Vector2i) -> Array[Vector2i]:
```

Comment:
This function doesn't explictly get the positions based on the player since we pass in the position in the arguments, so I think a better name would be 
`get_accessible_room_positions_by_given_position()`

Reply:
yes, will do